### PR TITLE
add copy button to submenu, universal copypaste component

### PR DESCRIPTION
**Pull request by fokkerone**
solves#1474. 
Should run Universal( Electron + Browser Figma)
Uses token.name as path value.

### CODE REVIEW

**Review by six7 — CHANGES_REQUESTED** [review]
Thank you for this.

Could you remove any codestyle changes? This makes it super hard to see what changed if everything changed.

File: src/app/components/MoreButton/MoreButton.tsx [hunk 1]
```diff
@@ -99,86 +93,71 @@ export const MoreButton: React.FC<Props> = ({
...
 
   return (
-    <ContextMenu>
-      <ContextMenuTrigger id={`${token.name}-button}`}>
+    <ContextMenu.ContextMenu>
+      <ContextMenu.ContextMenuTrigger id={`${token.name}-button}`}>
         <TokenButtonContent type={type} active={active} onClick={handleTokenClick} token={token} />
-      </ContextMenuTrigger>
-      <ContextMenuContent sideOffset={5} collisionTolerance={30}>
-        {visibleProperties.map((property) => (property.childProperties ? (
```
[the remark this answers is not in the document]
Comment (fokkerone) [reply src/app/components/MoreButton/MoreButton.tsx:124]: think its based on Next branch. I will rebase ist on Main branch. Whats the gitflow you are using? Branching from Next or Main?

**Comment by fokkerone** [comment]
> 



> Thank you for this.
> 
> Could you remove any codestyle changes? This makes it super hard to see what changed if everything changed.

I am using the TSConf of the project in VSCode. Think its based on my Prettier Config. Sorry. Try to adopt your codestyles.

File: src/app/components/MoreButton/MoreButton.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (six7) [reply src/app/components/MoreButton/MoreButton.tsx:124]: `next` preferred 👍 main is always what's in production

**Pull request closed without merging**